functions/openhab.bash

```diff
@@ -59,25 +59,25 @@ openhab_setup() {
   fi
 
   if [[ $2 == "unstable" ]]; then
```

Comment:
I recall this was changed before, why is this back ?? Did you maybe diff against some outdated git repo ?

Reply:
No, it outputs wrong the way it is, say your on my branch then it will output whatever the branch name is instead of openHAB (version 2 or three doesn't matter).